docs/ref/templates/builtins.txt

```diff
@@ -961,20 +961,20 @@ output (as a string) inside a variable. This is useful if you want to use
 
 Outputs a URL-encoded formatted query string based on the provided parameters.
 
-This tag requires a :class:`~django.http.QueryDict` instance, which defaults to
-:attr:`request.GET <django.http.HttpRequest.GET>` if none is provided.
+This tag requires a :class:`~django.http.QueryDict` or :class:`dict` instance,
+which defaults to :attr:`request.GET <django.http.HttpRequest.GET>`
+if none is provided.
 
-If the :class:`~django.http.QueryDict` is empty and no additional parameters
-are provided, an empty string is returned. A non-empty result includes a
-leading ``"?"``.
+If the ``dict``s are empty and no additional parameters are provided,
+an empty string is returned. A non-empty result includes a leading ``"?"``.
```

Comment:
I'm kind of thinking that the fact that the tag results to a `QueryDict` (right before `urlencode()`) is an implementation detail, and as far as I can tell we don't mention this elsewhere in the docs. It could seem a bit confusing, what do you think?

Reply:
I like this